NEWS.md

```diff
@@ -1,3 +1,24 @@
+# messydates 0.1.2
+
+## Package
+* Added PANARCHIC project details to README file
+* Added cheatsheet
+* Updated documentation
```

Comment:
You should either remove this line or be more specific about which documentation was updated